docs/scu/dev/how-the-spryker-code-upgrader-works.md

```diff
@@ -0,0 +1,60 @@
+---
+title: How the Spryker Code Upgrader works
+description: Spryker Code Upgrader overview
+template: concept-topic-template
+redirect_from:
+  - /docs/paas-plus/dev/how-the-spryker-code-upgrader-works.html
+---
+
+To update a project, the Spryker Code Upgrader runs the following steps.
+
+### 1. Identifies the Spryker modules to update
+
+The Spryker Code Upgrader performs the following sub-steps:
+
+1. To identify the modules to be updated, compares the information in the `composer.json` and `composer.lock` files with our latest released code.
```

Comment:
```suggestion
1. To identify the modules to be updated, it will compare the information present in the `composer.json` and `composer.lock` files with our latest released code.
```